tests/test_commands.py

```diff
@@ -234,6 +233,62 @@ def test_nickserv_and_memoserv(alice, bob, wait_until):
     wait_until(lambda: "send Bob hello there\n" in bob.text())
 
 
+@pytest.mark.skipif(
+    os.environ["IRC_SERVER"] == "hircd",
+    reason="hircd doesn't support away notifications",
+)
+def test_away_status(alice, bob, wait_until):
+    switch_to_channel_view(bob, "#autojoin")
+
+    alice.entry.insert("end", "/away foo bar baz")
+    alice.on_enter_pressed()
+
+    # Server view
+    wait_until(lambda: "You have been marked as being away\n" in alice.text())
```

Comment:
This in fact checks the channel view. If you want to check the server view, you can use `alice.get_server_views()[0].textwidget.get("1.0", "end")` instead of `alice.text()`. Or add a `.text()` method to `class View`, so that you can do simply `alice.get_server_views()[0].text()`.